packages/cashier/src/pages/payment-agent/payment-agent-card/payment-agent-card.tsx

```
@@ -1,20 +1,28 @@
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import React from 'react';
 import { ExpansionPanel } from '@deriv/components';
-import PaymentAgentCardDescription from './payment-agent-card-description.jsx';
+import PaymentAgentCardDescription from './payment-agent-card-description';
 import PaymentAgentDepositDetails from '../payment-agent-deposit-details';
 import PaymentAgentListedWithdrawForm from '../payment-agent-listed-withdraw-form';
 import './payment-agent-card.scss';
+import { TPaymentAgent } from '../../../types';
 
-const PaymentAgentCard = ({ is_dark_mode_on, is_deposit, payment_agent }) => {
+type TPaymentAgentCard = {
+    is_dark_mode_on?: boolean;
+    is_deposit?: boolean;
+    payment_agent: TPaymentAgent;
+};
+
+const PaymentAgentCard = ({ is_dark_mode_on, is_deposit, payment_agent }: TPaymentAgentCard) => {
     const message = {
```

@rostislav-deriv There is a type error on line 33.

---

Yes, I mentioned it [here](https://github.com/rostislav-deriv/deriv-app/pull/6#discussion_r1160704398), not sure how to fix it as of now 😓

---

@rostislav-deriv Oh I missed that comment, Sorry man 🌹 

Seems like the type in the `ExpansionPanel` is wrong, We can ignore the error in this PR and fix it later 🙇🏻

---

@heorhi-deriv helped with fixing it in terms of _using_ `ExpansionPanel`, but inside the `ExpansionPanel` itself there are a lot of problems with the typing for rendering nested arrays; it would require a lot of refactoring to align the types with the actual logic